src/routes.py

```diff
@@ -28,6 +32,9 @@
 api_users.add_resource(UsersResourceByUUID, '/uuid/<string:uuid>')
 api_users.add_resource(UsersResourceByUsername, '/username/<string:username>')
 api_users.add_resource(UsersVerifyResource, '/verify')
+api_users.add_resource(UserSiteResourceList, '/sites')
+api_users.add_resource(UserSiteResourceByUUID, '/sites/uuid/<string:uuid>')
+api_users.add_resource(UserSiteResourceByUserUUID, '/sites/user_uuid/<string:user_uuid>')
```

Comment:
Remove this one `UserSiteResourceByUserUUID`, I don't think we ever use this API.